lightkurve/lightcurve.py

```diff
@@ -1213,22 +1213,41 @@ def _create_plot(self, method='plot', ax=None, normalize=False,
                 xlabel = 'Time'
         # Default ylabel
         if ylabel is None:
-            if normalize or (self.flux.unit == u.dimensionless_unscaled):
-                ylabel = 'Normalized Flux'
-            elif self.flux.unit is None:
-                ylabel = 'Flux'
+            if column in ['flux','sap_flux','sap_bkg','pdscap_flux']:
+                if normalize or (self.flux.unit == u.dimensionless_unscaled):
+                    ylabel = 'Normalized Flux'
+                elif self.flux.unit is None:
+                    ylabel = 'Flux'
+                else:
+                    ylabel = 'Flux [{}]'.format(self.flux.unit.to_string("latex_inline"))
             else:
-                ylabel = 'Flux [{}]'.format(self.flux.unit.to_string("latex_inline"))
+                if normalize:
+                    ylabel = 'Normalized {}'.format(column)
+                elif self[column].unit is None:
+                    ylabel = '{}'.format(column)
+                else:
+                    ylabel = '{} [{}]'.format(column, self[column].unit.to_string("latex_inline"))
```

Comment:
Setting the default `ylabel` is becoming quite complicated.  I wonder if we could simplify roughly as follows:

```python
if ylabel is None:
    if "flux" in column:
        if normalize or self.meta.get("normalized"):
            ylabel = "Normalized Flux"
        else:
            ylabel = "Flux"
    else:
        ylabel = f"{column}"
    # Add unit if available
    if self[column].unit:
        ylabel += f" {self[column].unit.to_string("latex_inline")}"
```

Reply:
That's a really elegant solution! I've combined our two snippets of code into something that I think covers all the bases.